src/inline_snapshot/pytest_plugin.py

```diff
@@ -113,7 +113,9 @@ def pytest_configure(config):
 
         _inline_snapshot._update_flags = _inline_snapshot.Flags(flags & categories)
 
-    snapshot_path = Path(config.rootpath) / ".inline-snapshot/external"
+    snapshot_path = (
+        _config.config.snapshot_dir or config.rootpath / ".inline-snapshot/external"
+    )
```

Comment:
```suggestion
    snapshot_path = (
        _config.config.snapshot_dir or config.rootpath / ".inline-snapshot"
    )/"external"
```

That is what I mean. `snapshot-dir` will allow to change the `.inline-snapshot/` part. We could maybe change the name from `snapshot-dir` to `storage-dir`, because this is for now and in the future the place where inline-snapshot can store stuff. Maybe not just external snapshots.

Reply:
Oh OK, thanks, that was easier than I thought :slightly_smiling_face: 

OK so to summarize:

- we apply this suggestion
- we rename the setting to `storage-dir`
- we set its default to `.inline-snapshot`

Can you confirm? Anything else :slightly_smiling_face:?

Reply:
Done, pushed a fixup.

Reply:
:+1: thank you very much.